docs/Installation.md

```diff
@@ -0,0 +1,72 @@
+## Installing the Server
+
+Music Assistant can be operated as a complete standalone product but it is actually tailored to use side by side with Home Assistant, it is meant with automation in mind, hence our recommended installation method is to run the server as Home assistant Add-on which will be aitomatically installed when you add the Home Assistant Integration.
+
+MA requires a 64bit Operating System and a minimum of 2GB of RAM on the physical device or the container (physical devices are recommended to have 4GB+ if they are running anything else)
+
+### Primary installation method: Home Assistant Integration Installation
+
+Go to the [Home Assistant Integration](https://music-assistant.github.io/integration/) page and follow the instructions there which will install the integration and the server.
```

Comment:
Here I would use the internal linking rather than the weblink. Mkdocs should create the link correctly during build